src/profile_manager/urls.py

```diff
@@ -23,6 +23,7 @@
     url(r'^(?P<profile_id>[0-9]+)/comment_ban_toggle/$', views.comment_ban_toggle, name='comment_ban_toggle'),
     url(r'^(?P<profile_id>[0-9]+)/comment_ban/$', views.comment_ban, name='comment_ban'),
     url(r'^(?P<pk>[0-9]+)/edit/$', views.ProfileUpdate.as_view(), name='profile_update'),
+    url(r'^own/$', views.ProfileDetailOwn.as_view(), name='profile_detail_own'),
```

Comment:
Can you add basic access tests for this new view? For anonymous, random authenticated (verifying it is their own), and staff?

Reply:
added